test/k8sT/Policies.go

```diff
@@ -591,6 +605,82 @@ var _ = Describe("K8sValidatedPolicyTest", func() {
 			testConnectivitytoRedis()
 		})
 	})
+
+	Context("KubernetesNetworkPolicy between server and client", func() {
+		BeforeEach(func() {
+			kubectl = helpers.CreateKubectl(helpers.K8s1VMName(), logger)
+
+			By("Creating the namespace that will be used for the pods")
+			_, err := kubectl.CoreV1().Namespaces().Create(&v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: namespace}})
+			Expect(err).NotTo(HaveOccurred())
+
+			By("Creating a simple server that serves on port 80 and 81.")
+			podServer, service = createServerPodAndService(kubectl, namespace, "server", []int{80, 81})
+
+			By("Waiting for pod ready", func() {
+				ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+				defer cancel()
+				err := kubectl.WaitForPodReady(ctx, namespace, podServer.Name)
```

Comment:
Why you don't use the common pattern: 
```
			_, err := kubectl.WaitforPods(helpers.DefaultNamespace, "-l zgroup=testapp", 300)
```

Is only one line, and it's easier to understand.